mobile/examples/object_detection/ios/ORTObjectDetection/download.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# ============================================================
+
+MODELS_URL="https://tfhub.dev/tensorflow/lite-model/ssd_mobilenet_v1/1/metadata/1?lite-format=tflite"
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+DOWNLOAD_DIR="${SCRIPT_DIR}/ModelsAndData"
+
+mkdir -p ${DOWNLOAD_DIR}
+
+#Download source tflite model file
+curl -L ${MODELS_URL} >${DOWNLOAD_DIR}/ssd_mobilenet_v1_1_metadata_1.tflite
+
+#Unzip and get model metadata
+unzip ${DOWNLOAD_DIR}/ssd_mobilenet_v1_1_metadata_1.tflite -d ${DOWNLOAD_DIR}
+
+#Convert tflite model to onnx model
+python3 -m tf2onnx.convert --tflite ${DOWNLOAD_DIR}/ssd_mobilenet_v1_1_metadata_1.tflite --opset 13 --output ${DOWNLOAD_DIR}/ssd_mobilenet_v1.onnx
+
+#Convert onnx model to ort format
+python3 -m onnxruntime.tools.convert_onnx_models_to_ort ${DOWNLOAD_DIR}
```

Comment:
Would be good to mention in the comment what the output of this command is (the .ort file and its location) so the user has a hint of where to look and what they're looking for.